tests/component-themes-builder.php

```diff
@@ -4,27 +4,28 @@
 require_once( './server/class-component-themes.php' );
 
 // @codingStandardsIgnoreStart
-function component_Themes_Text_Widget( $props ) {
+function component_Themes_Text_Widget_Mock( $props ) {
 	// @codingStandardsIgnoreEnd
 	$class = ct_get_value( $props, 'className', '' );
 	$text = ct_get_value( $props, 'text', 'This is a text widget with no data!' );
 	$color = ct_get_value( $props, 'color', 'default' );
 	return React::createElement( 'div', [ 'className' => $class ], [ React::createElement( 'p', [], 'text is: ' . $text ), React::createElement( 'p', [], 'color is: ' . $color ) ] );
 };
-Component_Themes::register_component( 'TextWidget', 'Component_Themes_Text_Widget' );
 
-class Component_Themes_Column_Component extends Component_Themes_Component {
+class Component_Themes_Column_Component_Mock extends Component_Themes_Component {
 	public function render() {
 		return "<div class='" . $this->get_prop( 'className' ) . "'>" . $this->render_children() . '</div>';
 	}
 }
-Component_Themes::register_component( 'ColumnComponent', 'Component_Themes_Column_Component' );
 
 $test_partial = [ 'id' => 'helloWorld', 'componentType' => 'TextWidget', 'props' => [ 'text' => 'test partial' ] ];
 Component_Themes::register_partial( 'TestPartial', $test_partial );
 
 describe( 'Component_Themes_Builder', function() {
 	beforeEach( function( $c ) {
+		Component_Themes::register_component( 'TextWidget', 'Component_Themes_Text_Widget_Mock' );
```

Comment:
> re-registering multiple times wouldn't have any effect.

I know. But I had to move it into "beforeEach" block, or the real TextWidget component overwrote the global state so the test cases would call the real one. That would break the test.

Can you make Corretto framework execute each test file in a separated sandbox?